### PR TITLE
Improved keyboard accessibility of editor

### LINKED ISSUE #755: Keyboard accessibility issues
https://app.usebubbles.com/hkxWVuRNqqUfCL78prtNQC/keyboard-accessibility-issues-for-neeto-editor

### PR DESCRIPTION
**Pull request by gaagul**
- Fixes #755 

**Description**

- Updated: tabIndexes to improve the keyboard accessibility of editor and menu

**Checklist**

~~- [ ] I have made corresponding changes to the documentation.~~
~~- [ ] I have updated the types definition of modified exports.~~
- [x] I have verified the functionality in some of the neeto web-apps.
- [x] I have added the necessary label (patch/minor/major - If package publish
      is required).

**Reviewers**

@AbhayVAshokan

### CODE REVIEW

**Comment by gaagul** [comment]
@AbhayVAshokan _a The following video has the demo of new updates
https://vimeo.com/847292939/c2093a9831

**Review by AbhayVAshokan — CHANGES_REQUESTED** [review]
@gaagul _a check the feasibility of this: https://vimeo.com/847361272/803154733e

**Comment by gaagul** [comment]
> @gaagul _a check the feasibility of this

I will look into the possibilities of adding this feature.

**Comment by gaagul** [comment]
@AbhayVAshokan _a I have added arrow-key navigation for menu buttons Please review the changes.

File: src/components/Editor/Menu/Fixed/index.jsx [hunk 1]
```diff
@@ -44,10 +44,46 @@ const Fixed = ({
...
+  const menuRef = useRef(null);
+
+  const { t } = useTranslation();
+
+  const handleArrowNavigation = event => {
+    if (event.key === "ArrowRight") {
+      event.preventDefault();
+      setFocusedButtonIndex(prevIndex =>
+        prevIndex === -1 ? prevIndex + 2 : (prevIndex + 1) % menuButtons.length
+      );
```
Comment (AbhayVAshokan) [inline src/components/Editor/Menu/Fixed/index.jsx:60]:
```suggestion
      setFocusedButtonIndex(prevIndex => (prevIndex + 1) % menuButtons.length)
```

Can we get away with this implementation? For this to work, the default value in the `useState` should be 0.

**Review by AbhayVAshokan — CHANGES_REQUESTED** [review]
@gaagul _a

File: src/components/Editor/Menu/Fixed/index.jsx (hunk 1 above)
Comment (gaagul) [reply src/components/Editor/Menu/Fixed/index.jsx]: I will work on that, This might be possible once i update the useEffect.

**Comment by gaagul** [comment]
@AbhayVAshokan _a Please verify the new changes.

**Review by AbhayVAshokan — APPROVED** [review]

**Pull request merged**